### ISSUE TITLE
Portfolio - search and grouping does not work together

### ISSUE THREAD

**Issue by tarjeieo** [Puzzlepart/prosjektportalen#691]
Thank you for reporting an issue. We appreciate your feedback - to help the team understand your needs please complete the below template to ensure we have the details to help. Thanks!

#### Environment

[X] SharePoint Online

[ ] SharePoint 2013

[ ] SharePoint 2016

#### Browser

[X] Chrome

[ ] Internet Explorer

[ ] Edge

[ ] Safari

[ ] Other

#### Version

Please specify what version you are using: [ 2.5.14]

#### Summary

If you group and search at the same time, the projects are seemingly scrambled random.

It doesn't matter if you search first or group first.

**Comment by olemp** [comment]
Probably the grouping doesn't recalculate after the search text changes @Remi749? Shouldn't be a need for a new function.

**Comment by Remi749** [comment]
@olemp that is correct, is there functionality for recalculating this? If so, what?

**Comment by olemp** [comment]
Check `getFilteredData`: https://github.com/Puzzlepart/prosjektportalen/blob/master/src/js/WebParts/DynamicPortfolio/index.tsx#L411
